components/product/ProductCard.js

```diff
@@ -21,21 +22,29 @@ function ProductCard({
           store,
         })
       }>
-      <ColumnContainer>
-        {productsScreen ? (
-          <Image
-            source={{ uri: product.imageUrl }}
-            style={{ height: 86, width: 151, borderRadius: 12 }}
-          />
-        ) : (
-          <Image
-            source={{ uri: product.imageUrl }}
-            style={{ height: 86, width: 86, borderRadius: 12 }}
-          />
-        )}
-        <Body>{product.name}</Body>
+      <ColumnContainer
+        style={
+          productsScreen
+            ? { width: (Window.width - 32 - 40) / 2 }
+            : { width: 86 }
+        }>
+        <Image
+          source={{ uri: product.imageUrl }}
+          style={{
+            height: 86,
```

Comment:
I have been bewildered by the formatting changes from the last couple PRs, and just realized that eslint is picky about this too so it auto-fixes it.. 😓 We might want to remove the `--fix` from the pre-commit hook and instead just run `eslint`. I'll probably update this in my linting PRs